benchmarks/htap/schemas/common/new_order.sql

```diff
@@ -9,28 +9,8 @@ CREATE FUNCTION new_order(
   , in_supware INT[]
   , in_qty INT[]
   , in_timestamp TIMESTAMPTZ
-) RETURNS TABLE(
```

Comment:
can we do this also with the other functions?

Reply:
So `order_status` and `payment` can be `VOID` same as `delivery`, right?

Reply:
i think so yes?

Reply:
Done